xtend-contrib/build.gradle

```diff
@@ -1,20 +1,24 @@
 plugins {
-	id 'org.xtend.xtend' version '0.4.5'
-	id 'com.github.oehme.sobula.bintray-release' version '0.4.1'
-	id 'com.github.oehme.sobula.stefan' version '0.4.1'
+	id 'org.xtext.xtend' version '1.0.3'
+	id 'com.github.oehme.sobula.bintray-release' version '0.6.6'
+	id 'com.github.oehme.sobula.stefan' version '0.6.6'
 }
 
 group = 'com.github.oehme.xtend'
 description = 'Automates well known Java Patterns using Xtend Active Annotations'
 repositories.jcenter()
 
+sourceSets {
+	main.xtendOutputDir = 'src/main/xtend-gen'
```

Comment:
- .gitignore entries are missing
- a small comment explaining why we do this (so we can remove it when Buildship does the right thing)
- move the duplication to a root build.gradle script